packages/edit-site/src/plugins/keyboard-shortcut-help-menu-item.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useReducer } from '@wordpress/element';
+import { useShortcut } from '@wordpress/keyboard-shortcuts';
+import { displayShortcut } from '@wordpress/keycodes';
+import { MenuItem } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import KeyboardShortcutHelpModal from '../components/keyboard-shortcut-help-modal';
+
+export default function KeyboardShortcutsHelpMenuItem() {
+	const [ isModalActive, toggleModal ] = useReducer(
+		( isActive ) => ! isActive,
+		false
+	);
+
+	useShortcut( 'core/edit-site/keyboard-shortcuts', toggleModal );
```

Comment:
This makes the shortcut only available when we have this menu item rendered, that is, when the options dropdown is opened. This is not the case in the post editor (or any other editors) though, hence I believe it should be a bug?

Reply:
Good catch, @kevin940726. I'll have to do some refactoring on how we render "Tool Groups" to achieve this.

Reply:
The issue should be fixed via f11124a.

* I'm rendering the "Tools" menu group directly in the "MoreMenu" component instead of using the plugin and SlotFill.
* The `ToolsMoreMenuGroup` still can be used by third parties to render items inside the "Tools" menu group.